packages/ui/src/forms/RenderFields/RenderField.tsx

```diff
@@ -85,6 +85,19 @@ export const RenderField: React.FC<Props> = ({
       />
     )
   } else {
+    if (fieldComponentProps.field.type === 'row') {
+      for (const field of fieldComponentProps.field.fields) {
+        if (field.admin?.width) {
+          field.admin.style = {
+            ...field.admin.style,
+            maxWidth: field.admin.width,
```

Comment:
set `max-width` CSS property based on `field.admin.width` if we are in a `row` field